### ISSUE TITLE
Update for python 3 compatibility

### ISSUE THREAD

**Issue by jaitaiwan** [livewires/python#2]
Hi Guys,

I'm a programmer thanks to going to a Scripture Union camp in QLD (Ubertweak) and going through your livewires course. I've now come full circle and I'm wanting to use your course to teach people to program using livewires.

I started this issue to address a few things:
- Would you be open to allowing me to contribute to the project to update it for py3k compatibility?
- If so, would you be open to changing the licence to one which encouraged this sort of work?
- Would I be allowed to use the name livewires when advertising the course for students?

If the answer to these questions are yes, then I'll fork the project and begin a pull request to track all the python 3 changes. If I'm feeling up to it, I may include unittests to automatically run against the project so that future modifications and additions can be assured of their stability and backwards compatibility.

Cheers

**Comment by jaitaiwan** [comment]
Hey Rob and others,

I've nearly completed updating not only the code, but all the sheets. Sheets 1-5 are updated, and I'm up to sheet D of R.

Along the way I've tested practically every line of code in the example and I've nearly run over all the code in the beginners module.

Some changes that you may not want to pull across:

- Changed any instance of the word Colour to Color (rationale below)
- Updated the module name to "pieisreal" (so as not to infringe on the copyright)
- Updated urls for the project location

## A note about compatibility
I made the decision not to support both python 2 and 3 when updating the library because:

1. Py3k is inevitable with Python 2.7 the intended last release of python (admittedly with extended support till 2020).
2. Py3k is available by default on most modern distributions of linux.
3. It adds complexity to the library which has the potential to confuse any students who are intending on reading some of the library to understand how things work.

## Colour vs Color
Understandably, having been a primarily UK English course you'd want to stick with "Colour". However for the majority of work that I do in the web sphere, it requires working in US English. I'm perfectly capable of writing both depending on the situation, but it does become a frustration in CSS when writing Colour and spending a good few seconds figuring out why your change didn't work. There are other places where this happens as well, for instance GTK theming.

Obviously this reasoning may not apply to everyone, but if it helps you to cherry pick less patches from my fork then all the better :)

## In Summary
Updates to the library are coming, but will need to be merged with patches rather than a straight merge. You can see progress here <http://github.com/centricwebestate/pieisreal>

**Comment by jaitaiwan** [comment]
I should also mention that I haven't updated anything other than the beginners because until recently, pip didn't parse the pygame website correctly when working out what versions can be installed. Specifically version 1.9 is needed for py3k compatibility.

The only way to install pygame at this time for py3k is this command:
`pip install pygame==1.9.1release --allow-external pygame --allow-unverified pygame`

Which for now is unacceptable for me.
